arbnode/espresso/submitter/original_submitter.go

```diff
@@ -0,0 +1,779 @@
+package submitter
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"math/big"
+	"os"
+	"sync"
+	"time"
+
+	tagged_base64 "github.com/EspressoSystems/espresso-network/sdks/go/tagged-base64"
+	espresso_types "github.com/EspressoSystems/espresso-network/sdks/go/types"
+	"github.com/hf/nitrite"
+	"github.com/hf/nsm"
+	"github.com/hf/nsm/request"
+
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethdb"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/rlp"
+
+	"github.com/offchainlabs/nitro/arbnode/espresso"
+	espresso_key_manager "github.com/offchainlabs/nitro/arbnode/espresso/key-manager"
+	"github.com/offchainlabs/nitro/arbos/arbostypes"
+	"github.com/offchainlabs/nitro/arbutil"
+	"github.com/offchainlabs/nitro/util"
+	"github.com/offchainlabs/nitro/util/dbutil"
+	"github.com/offchainlabs/nitro/util/stopwaiter"
+)
+
+var (
+	// Schema extensions for Espresso Tracking
+	espressoSubmittedTxns        []byte = []byte("_espressoSubmittedTxns")    // contains the hash and pos of the submitted transactions
+	espressoPendingTxnsPositions []byte = []byte("_espressoPendingTxnsPos")   // contains the index of the pending txns that need to be submitted to espresso
+	espressoLastConfirmedPos     []byte = []byte("_espressoLastConfirmedPos") // contains the position of the last confirmed message
+)
+
+type EspressoOriginalSubmitter struct {
```

Comment:
nit: same here, not sure what original means in this context? should prob rename

Reply:
This is named `EspressoOriginalSubmitter`, as I've made the effort to extract this submission to Espresso process into it's own struct.  It is named `OriginalSubmitter`, as it reflects the current implementation / strategy with no real logical modifications.

The purpose of this is to compare it with different strategies within the same test, so one might benchmark both.